### ISSUE TITLE
[feat.]: Index statistics

### ISSUE THREAD

**Issue by yaochen1205** [oceanbase/oceanbase#1378]
### Please check the issue list for the same feature first

- [X] Please checked issue list and confirm this feature is encountered for the first time.
- [X] Please try full text in English and attach precise description.

### A full description of the feature and provide some example if the same feature exists in other products, such as mysql

```markdown
To increase the statistics for indexing, a view or table can be created that records the number of reads and writes.  Similar to the MySQL performance_schema. table_io_waits_summary_by_index_usage, this method can be used to determine the usage of indexes and tables.  By tracking the number of reads and writes for each index, it is possible to determine if a table is still being accessed, and the frequency of access.
```

**Comment by hnwyllmm** [comment]
We will do it later, thanks for your suggestion.

**Comment by hnwyllmm** [comment]
closed by 143a6f2f2e1844b3a3d40416ae0d24e81279a8ba
